lib/doorkeeper/oauth/token.rb

```diff
@@ -55,7 +55,11 @@ def self.from_request(request, *methods)
 
       def self.authenticate(request, *methods)
         if token = from_request(request, *methods)
-          AccessToken.by_token(token)
+          access_token = AccessToken.by_token(token)
+          if access_token && access_token.previous_refresh_token.present?
+            access_token.revoke_previous_refresh_token!
```

Comment:
We don't need it if we don't use the confditional in https://github.com/doorkeeper-gem/doorkeeper/pull/691/files#diff-aa3b637335d18183d5e2717f3b25689aR64